pkg/label/label.go

```diff
@@ -0,0 +1,8 @@
+// Package label contains common Kubernetes object labels. These are defined in
+// https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md
```

Comment:
Full stop please.

Reply:
Just to verify: Full stop after link as well? In general that would break support for directly clicking link in editor.

Reply:
I rather see this is a sentence which contains a link. 

```
These are defined in <link>.
```

Reply:
Towards breaking links in editors, I personally do not have this feature.